Apps/WebClient/src/NewClientApp/src/views/HomeView.vue

```diff
@@ -1,7 +1,541 @@
+}
+
+watch(vaccineRecordState, () => {
+    if (vaccineRecordState.value.resultMessage.length > 0) {
+        // vaccineRecordResultModal.value?.showModal(); // TODO: Reinstate
+    }
+
+    if (
+        vaccineRecordState.value.record !== undefined &&
+        vaccineRecordState.value.status === LoadStatus.LOADED &&
+        vaccineRecordState.value.download
+    ) {
+        const mimeType = vaccineRecordState.value.record.document.mediaType;
+        const downloadLink = `data:${mimeType};base64,${vaccineRecordState.value.record.document.data}`;
+        fetch(downloadLink).then((res) => {
+            SnowPlow.trackEvent({
+                action: "click_button",
+                text: "FederalPVC",
+            });
+            res.blob().then((blob) => saveAs(blob, "VaccineProof.pdf"));
+        });
+        stopAuthenticatedVaccineRecordDownload(user.value.hdid);
+    }
+});
+</script>
 
 <template>
-    <h1>home</h1>
+    <v-container>
+        <LoadingComponent
+            :is-loading="isVaccineRecordDownloading"
+            :text="vaccineRecordStatusMessage"
+        />
+        <v-alert
+            v-if="unverifiedEmail || unverifiedSMS"
+            data-testid="incomplete-profile-banner"
+            show
+            dismissible
+            type="info"
+            class="no-print my-3"
+            title="Verify Contact Information"
+        >
+            <span class="text-body-1">
+                Your email or cell phone number has not been verified. You can
+                use the Health Gateway without verified contact information,
+                however, you will not receive notifications. Visit the
+                <router-link
+                    id="profilePageLink"
+                    data-testid="profile-page-link"
+                    variant="primary"
+                    to="/profile"
+                    >Profile Page</router-link
+                >
+                to complete your verification.
+            </span>
+        </v-alert>
+        <v-alert
+            v-if="!isPacificTime"
+            show
+            closable
+            type="info"
+            title="Looks like you're in a different timezone."
+            text="Heads up: your health records are recorded and displayed in
+                Pacific Time."
+            class="no-print my-3"
+        />
+        <PageTitleComponent title="Home">
+            <template #append>
+                <AddQuickLinkComponent
+                    :disabled="isAddQuickLinkButtonDisabled"
+                />
+            </template>
+        </PageTitleComponent>
+        <v-row>
+            <v-col :cols="quickLinkCols">
+                <HgCardComponent
+                    title="Health Records"
+                    data-testid="health-records-card"
+                    @click="handleClickHealthRecords()"
+                    class="h-100"
+                    elevation="6"
```

Comment:
(Applies throughout.)

Suggest swapping to new suggested way of maximizing card height using `class="d-flex"` on the `<v-col>` and `class="flex-grow-1"` on the `<HgCardComponent>`.

Also suggest removing the extra elevation for now & seeing how default card styling is received.

One last thing - our linting should be complaining that there are regular props underneath the `@click` event handler.  Event handlers should be listed last.